packages/data-context/src/DataContext.ts

```diff
@@ -154,7 +154,7 @@ export class DataContext {
   }
 
   async versions () {
-    return new VersionsDataSource().versions()
+    return new VersionsDataSource(this).versions()
```

Comment:
Mentioned this while meeting w/ @ryanthemanuel - we should be caching `versions` on the class

```
@cached
get versions() {
  return new VersionsDataSource(this)
}
```

and then elsewhere do `ctx.versions.getVersions()` or something

Reply:
Made this change:

[`19b094d` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/19b094d70148ddc58e68b5c224f33e940137306a)

[`19b094d` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/19b094d70148ddc58e68b5c224f33e940137306a)